### PR TITLE
[DDW-522] Display fee and deposit on transaction details and delegation modal

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR displayed fee and deposit info on transaction details and delegation confirmation modal
[Jira Ticket](https://jira.iohk.io/browse/DDW-522)

## Todos
- [x] Display fee and deposit info on transaction details panel
- [x] Display fee and deposit info on delegation confirmation modal
- [x] Display fee and deposit info on transaction details panel also for Hardware Wallets
- [x] Display fee and deposit info on delegation confirmation modal also for Hardware Wallets

## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1612427519147700)

## Test Cases

```Gherkin
Background:
Given Daedalus language is (English|Japanese)

Scenario 1: Delegating wallet for the first time
Given I have a wallet with enough ada for staking
And I am at delegation process
When I am at ‘Confirm Delegation’ step
Then I could see the calculated fees displayed
And the calculated deposit is displayed as 2.000000 ADA

Scenario 2: Transaction after delegating wallet for the first time
Given I have successfully delegated a wallet for the first time
When I go to Transaction tab
And I check the displayed transaction
Then I could see the calculated Transaction fee displayed
And the calculated deposit is displayed as 2.000000 ADA

Scenario 3: Redelegating wallet
Given I have a wallet with enough ada for staking
And the wallet has been delegated before
And I am at redelegation process
When I am at ‘Confirm Delegation’ step
Then I could see the calculated fees displayed
And the calculated deposit is not displayed

Scenario 4: Transaction after redelegating wallet
Given I have successfully redelegated a wallet
When I go to Transaction tab
And I check the displayed transaction
Then I could see the calculated Transaction fee displayed
And there is no deposit displayed
```

### App
- [x] Go to `Wallet` => `Transactions` and expand transaction items and make sure `fee` is shown in any outgoing or delegation transaction item and `deposit` is shown in any delegation or undelegation transaction item
- [x] Go to `Staking` => `Stake pools` => `Delegation` wizard and continue to confirmation step and make sure `fee` and `deposit` values are shown

### English
image

### Japanese
image

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by mioriohk — APPROVED** [review]
LGTM. Good job @yakovkaravelov 👍 . Tested on 16267

**Review by ManusMcCole — APPROVED** [review]

**Comment by nikolaglumac** [comment]
@a-rukin please test this one

**Review by a-rukin — CHANGES_REQUESTED** [review]

File: source/renderer/app/themes/daedalus/incentivized-testnet.js [hunk 1]
```diff
@@ -923,6 +923,7 @@ export const INCENTIVIZED_TESTNET_THEME_OUTPUT = {
     '--theme-transactions-list-border-color': '1e1f31',
     '--theme-transactions-list-group-date-color': '#ffffff',
     '--theme-transactions-list-item-details-color': '#ffffff',
+    '--theme-transactions-list-item-highlight-color': '#ea4c5b',
```
Comment (nikolaglumac) [inline source/renderer/app/themes/daedalus/incentivized-testnet.js:926]: @yakovkaravelov please change this colour to `#ea4c5b `

**Review by nikolaglumac — CHANGES_REQUESTED** [review]

**Review by nikolaglumac — APPROVED** [review]

**Comment by nikolaglumac** [comment]
@yakovkaravelov please add screenshots of the delegation wizard which includes the deposit 🙏

**Pull request merged**